Rakefile

```diff
@@ -25,10 +21,21 @@ file 'README.md' => ['lib/debug/session.rb', 'lib/debug/config.rb',
   puts 'README.md is updated.'
 end
 
-task :test_protocol do
-  ENV['RUBY_DEBUG_PROTOCOL_TEST'] = '1'
+desc "Run all debugger console related tests"
+Rake::TestTask.new(:test_debug) do |t|
+  t.test_files = FileList["test/debug/*_test.rb", "test/*_test.rb"]
 end
 
+desc "Run all debugger procotols (CAP & DAP) related tests"
 Rake::TestTask.new(:test_protocol) do |t|
   t.test_files = FileList["test/protocol/*_test.rb"]
 end
+
+task :test do
+  fail <<~MSG
+    `rake test` isn't supported yet because protocol tests are still unstable.
+    To run all test cases, please use `rake test_all` instead.
+  MSG
```

Comment:
@ono-max Instead of just a comment, I think explicitly telling the users that `rake test` isn't supported yet would be better. wdyt?

Reply:
I'm ok with that 👍